pkg/service/http_service.go

```diff
@@ -73,15 +73,16 @@ func NewHttpServer(host string, port int, db storage.DB, jobManager *ccr.JobMana
 
 type CreateCcrRequest struct {
 	// must need all fields required
-	Name      string    `json:"name,required"`
-	Src       base.Spec `json:"src,required"`
-	Dest      base.Spec `json:"dest,required"`
-	SkipError bool      `json:"skip_error"`
+	Name        string    `json:"name,required"`
+	Src         base.Spec `json:"src,required"`
+	Dest        base.Spec `json:"dest,required"`
+	SkipError   bool      `json:"skip_error"`
+	ExceptTable string    `json:"except_table"`
```

Comment:
Define it to an array might be better.

```suggestion
	ExceptTables []string    `json:"except_tables"`
```

Reply:
Then run `strings.TrimSpace(exceptTable)` before converting `jobContext` to `Job`.